picketlink/src/main/java/org/wildfly/extension/picketlink/federation/model/idp/TrustDomainResourceDefinition.java

```diff
@@ -74,7 +74,7 @@ protected void validateUpdatedModel(OperationContext context, Resource model) th
     public static void validateModelInOperation(OperationContext context, ModelNode model) throws OperationFailedException {
         String certAliasAttribName = ModelElement.IDENTITY_PROVIDER_TRUST_DOMAIN_CERT_ALIAS.getName();
 
-        if (context.getProcessType().isServer() && model.hasDefined(certAliasAttribName)) {
+        if (model.hasDefined(certAliasAttribName)) {
```

Comment:
Instead of removing the isServer() check, shouldn't the whole block just go away since this will never be a server?